### PR TITLE
Add portuguese from Brasil translation.

### PR DESCRIPTION
**Pull request by AlexandreSato**
Missing .qm file

### CODE REVIEW

**Comment by AlexandreSato** [comment]
Shane, added th_TH translation by tape#7233 (spreadsheet method)

File: selfdrive/ui/translations/languages.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "English": "",
+  "Português": "main_pt",
+  "Thai": "main_th",
```
Comment (adeebshihadeh) [inline selfdrive/ui/translations/languages.json:4]: each language should go in its own PR
